### ISSUE TITLE
[Request & Bug?] History

### ISSUE THREAD

**Issue by Lustsifer** [LagradOst/QuickNovel#26]
A more detailed history will be appreciated. For example, if we read a novel to chapter 23 then the history will display it at chapter 23 not the total chapter. 

![Screenshot_20210918-104005](https://user-images.githubusercontent.com/84452966/133871480-7fd4920c-91d6-411c-be07-d54a6048baa5.png)
![Screenshot_20210918-103941](https://user-images.githubusercontent.com/84452966/133871473-aaf1813f-66e0-48d3-a60c-de1eaa892a6e.png)

And I find that when I delete a novel from history it just delete that novel from history but not the reading progress, is it intended to function that way? If not, can you make it when we delete a novel from history the reading progress will be deleted too?

**Comment by LagradOst** [comment]
Yes, this is the intended feature. It only removes the visibility in the history as I treat the built in reader as another app. I could remove the reading progress too, but I dont see the need or reason to do so.

**Comment by Lustsifer** [comment]
Removing the reading progress can help in case like my issue #27 happen. If that is not in your todo list then do you have an alternative way to clear reading progress?

**Comment by LagradOst** [comment]
@Lustsifer do you want to clear the read progress to skip the invalid chapter?  Because atm there are no way to clear or remove it.

**Comment by Lustsifer** [comment]
@LagradOst Yeah, I want to skip the invalid chapter to continue reading if I can.
